packages/cli/src/cmds/validator/keys.ts

```diff
@@ -59,6 +59,17 @@ export async function getSecretKeys(args: IValidatorCliArgs & IGlobalArgs): Prom
   }
 }
 
+export async function getPublicKeys(args: IValidatorCliArgs & IGlobalArgs): Promise<PublicKey[]> {
+  const pubkeys: PublicKey[] = [];
+  const accountPaths = getAccountPaths(args);
+  const file = fs.readFileSync(accountPaths.publicKeysFile, "utf8");
+  const arr = file.toString().replace(/\r\n/g, "\n").split("\n");
```

Comment:
Why is the `.toString()` necessary here?

Reply:
Instead of replacing `\r\n` with `\n` then spiting by `\n` you can split by `\r\n` directly.

Reply:
This will be removed if we get our public keys from cli or yaml file.